eng/Versions.props

```diff
@@ -7,8 +7,8 @@
   <PropertyGroup>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>5</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>6</PreReleaseVersionLabel>
+    <PatchVersion>2</PatchVersion>
```

Comment:
Does this correspond to 17.5.**2**? What does `<PreReleaseVersionLabel>3</PreReleaseVersionLabel>` correspond to?

Reply:
I believe PreReleaseVersionLabel is used for preview package versions. Now that 17.5 GA is out, that number is irrelevant